openpype/hosts/maya/api/plugin.py

```diff
@@ -153,6 +155,25 @@ def load(
                 suffix="_",
             )
 
+            custom_naming = settings['maya']['load']['reference_loader']['naming']  # noqa
```

Comment:
Should this logic for custom naming be above the 'add a counter suffix' above?
What happens with custom naming enabled if you were to load e.g. a pointcache with count = 5? They'd each get the same namespace assigned since there's nothing ensuring uniqueness. Seems breaking?

Reply:
I'm not sure what you are saying here. All I can tell you is that if multiple references are loaded with the same namespace, Maya will automatically add a number at the end of the namespace to ensure uniqueness.

Reply:
Different references should have a unique namespace even if referencing the same file. The custom naming logic doesn't ensure the namespace is unique and thus problems will ensue.